docs/configuration.md

```diff
@@ -0,0 +1,158 @@
+---
+id: config
+title: Dapplet`s config
+---
+
+When creating a dapplet module, one of the important entities is the **dapplet's config**. It allows you to create and configure individual parameters for the module.
+
+For example: the network in which the dapplet will work, configure the monetization of the dapplet, change the url addresses of the server and the overlay part of the dapplet.
+
+The configuration is implemented through the `schema.json`.
+
+The schema follows the rules defined in http://json-schema.org/.
+The Dapplet settings UI is generated by [react-jsonschema-form](https://react-jsonschema-form.readthedocs.io/en/latest/usage/single/).
+
+In the production environment, we set up the **dapplet's config**, and specify the parameters that will be available in the browser through the extension.
+
+:::caution
+
+It is possible to edit and create a **dapplet's config** only from the working environment of the module in the extension, it is possible to use the **dapplet's config**, depending on how the developer configured them.
+
+:::
+
+
+### Configuration
+
+The **dapplet's config** is located in the */config* folder.
+
+In the `schema.json` file, specify the dapplet settings that will be available to the user.
+
+The scheme consists of parameters:
+
+  ```js
+ {
+// schema type: always object
+  "type": "object",
+// the fields you want to specify in your schema
+  "required": [
+   ...
+  ],
+// field properties
+  "properties": {
+   ...
```

Comment:
Not only properties but the entire parameters. I think that it's more convenient to unite this block with the next one.